set.mm

```diff
@@ -79193,22 +79191,22 @@ currently used conventions for such cases (see ~ cbvmpox , ~ ovmpox and
        19-Jun-2012.)  (Revised by Mario Carneiro, 14-Nov-2014.) $)
     tz7.44-2 $p |- ( suc B e. X -> ( F ` suc B ) = ( H ` ( F ` B ) ) ) $=
       ( wcel cfv c0 wceq cuni fveq2d cvv csuc cres cdm wlim crn cv fveq2 reseq2
-      cif eqeq12d vtoclga eleq1d noel dmeq dm0 syl6eq con0 word wss ordsson wtr
-      ax-mp ordtr trsuc mpan sseldi sucidg syl cin dmres ordelss fndm sseqtrrdi
-      wfn df-ss sylib syl5eq eleqtrrd eleq2 syl5ibcom syl5 iffalsed wn nlimsucg
-      mtoi wb limeq mtbird unieqd eloni ordunisuc 3syl eqtrd fvresd 3eqtrd fvex
-      syl6eqel eqeq1 rneq fveq1 ifbieq12d ifbieq2d fvmptg syl2anc ) DUAZHNZXEEO
-      ZEXEUBZFOZXHPQZCXHUCZUDZXHUEZRZXKRZXHOZGOZUIZUIZDEOZGOZBUFZEOZEYBUBZFOZQX
-      GXIQBXEHYBXEQZYCXGYEXIYBXEEUGYFYDXHFYBXEEUHZSUJJUKXFXHTNZXSTNXIXSQYDTNYHB
-      XEHYFYDXHTYGULKUKXFXSYATXFXSXRXQYAXFXJCXRXFXJDPNZDUMXJXKPQZXFYIXJXKPUCPXH
-      PUNUOUPXFDXKNYJYIXFDXEXKXFDUQNZDXENXFHUQDHURZHUQUSMHUTVBHVAZXFDHNYLYMMHVC
-      VBHDVDVEVFZDUQVGVHZXFXKXEEUCZVIZXEEXEVJXFXEYPUSYQXEQXFXEHYPYLXFXEHUSMHXEV
-      KVEEHVNYPHQLHEVLVBVMXEYPVOVPVQZVRXKPDVSVTWAWEWBXFXLXNXQXFXLXEUDZXFYKYSWCY
-      NDUQWDVHXFXKXEQXLYSWFYRXKXEWGVHWHWBXFXPXTGXFXPDXHOXTXFXODXHXFXOXERZDXFXKX
-      EYRWIXFYKDURYTDQYNDWJDWKWLWMSXFDXEEYOWNWMSWOZXTGWPWQAXHAUFZPQZCUUBUCZUDZU
-      UBUEZRZUUDRZUUBOZGOZUIZUIXSTTFUUBXHQZUUCXJUUKXRCUUBXHPWRUULUUEXLUUGUUJXNX
-      QUULUUDXKQUUEXLWFUUBXHUNZUUDXKWGVHUULUUFXMUUBXHWSWIUULUUIXPGUULUUIUUHXHOX
-      PUUHUUBXHWTUULUUHXOXHUULUUDXKUUMWISWMSXAXBIXCXDUUAWO $.
+      cif eqeq12d vtoclga eqeq1 wb dmeq limeq rneq unieqd fveq1 eqtrd ifbieq12d
+      syl ifbieq2d eleq1d noel dm0 syl6eq con0 word wss ordsson ax-mp wtr ordtr
+      trsuc mpan sseldi sucidg cin dmres ordelss wfn fndm sseqtrrdi df-ss sylib
+      syl5eq eleqtrrd syl5ibcom syl5 mtoi iffalsed wn nlimsucg mtbird ordunisuc
+      eleq2 eloni 3syl fvresd 3eqtrd fvex syl6eqel fvmptd3 ) DUAZHNZXDEOZEXDUBZ
+      FOZXGPQZCXGUCZUDZXGUEZRZXJRZXGOZGOZUIZUIZDEOZGOZBUFZEOZEYAUBZFOZQXFXHQBXD
+      HYAXDQZYBXFYDXHYAXDEUGYEYCXGFYAXDEUHZSUJJUKXEAXGAUFZPQZCYGUCZUDZYGUEZRZYI
+      RZYGOZGOZUIZUIXRTFTIYGXGQZYHXIYPXQCYGXGPULYQYJXKYLYOXMXPYQYIXJQYJXKUMYGXG
+      UNZYIXJUOVAYQYKXLYGXGUPUQYQYNXOGYQYNYMXGOXOYMYGXGURYQYMXNXGYQYIXJYRUQSUSS
+      UTVBYCTNXGTNBXDHYEYCXGTYFVCKUKXEXRXTTXEXRXQXPXTXEXICXQXEXIDPNZDVDXIXJPQZX
+      EYSXIXJPUCPXGPUNVEVFXEDXJNYTYSXEDXDXJXEDVGNZDXDNXEHVGDHVHZHVGVIMHVJVKHVLZ
+      XEDHNUUBUUCMHVMVKHDVNVOVPZDVGVQVAZXEXJXDEUCZVRZXDEXDVSXEXDUUFVIUUGXDQXEXD
+      HUUFUUBXEXDHVIMHXDVTVOEHWAUUFHQLHEWBVKWCXDUUFWDWEWFZWGXJPDWPWHWIWJWKXEXKX
+      MXPXEXKXDUDZXEUUAUUIWLUUDDVGWMVAXEXJXDQXKUUIUMUUHXJXDUOVAWNWKXEXOXSGXEXOD
+      XGOXSXEXNDXGXEXNXDRZDXEXJXDUUHUQXEUUADVHUUJDQUUDDWQDWOWRUSSXEDXDEUUEWSUSS
+      WTZXSGXAXBXCUUKWT $.
 
```

Comment:
The proof shrinks from 74 to 73 lines.  No change in Syntax, Axioms and Definitions usage.